pmda/rmsf.py

```diff
@@ -203,16 +203,18 @@ def _reduce(res, result_single_frame):
         atoms, mean, sumsq = result_single_frame
         positions = atoms.positions.astype(np.float64)
         # initial time step case
-        if len(res) == 0:
+        if isinstance(res, list) and len(res) == 0:
```

Comment:
Was this the problem? Why did this lead to an error for the special case?

If this was the problem then I am glad you found it. I'd like to know if this issue was specific to your case or does this affect other "aggregate" analysis classes, too, i.e., is the way that we work with the return values very fragile or at least too poorly documented? Does @VOD555 have any insights here?